services/site/tests/components/challenge/level/CodeLevel.spec.tsx

```diff
@@ -95,40 +89,38 @@ describe("Code Level", () => {
     router.back = jest.fn();
     jest.restoreAllMocks();
     (useSessionState as jest.Mock).mockImplementation(() => [1, jest.fn()]);
+    (useFlashMessageApi as jest.Mock).mockImplementation(() => ({
+      show: jest.fn(),
+      hide: jest.fn(),
+    }));
   });
 
   afterEach(() => {
     jest.resetAllMocks();
   });
 
-  it("renders `Sidebar` component", () => {
-    const view = mountCodeLevel();
+  it("renders instructions", () => {
```

Comment:
I would add "(Sidebar)" to the test description so it is more understandable which instructions are meant.